### DISCUSSION TITLE
Consider reverting back to the previous behavior when choosing to practice missed and slow words

### DISCUSSION THREAD

**Discussion by userrand** [monkeytypegame/monkeytype#5753] [Ideas]
The monkeytype website is great for practicing typing. I would like to suggest adding a setting or reverting back to a previous behavior which I explain below.

 It was possible before to practice missed and slow words without using the mouse by pressing esc for the command pallete then searching for "practice both" for example. Now it seems one can only search for "practice" then after pressing enter, a popup appears which either requires using the mouse to start practicing or using tab multiple times. Using the mouse is a bit tedious for me, the previous behavior where it was possible to practice using only the command pallete was nicer to me.

**Comment by fehmer** [comment]
I agree. open a popup from command line is strange.

**Comment by Leonabcd123** [comment]
#7125
